src/components/ResultContainer/ResultContainer.tsx

```diff
@@ -31,9 +33,12 @@ export default function ResultContainer(props: IResultContainerProps) {
       return (
         <div className='cio-results-container'>
           <h1 className='cio-results-title'>{resultsTitle}</h1>
-          <div className='cio-results-filter-and-redo-container'>
```

Comment:
Good call, 3ac51fa9ddcf71f28e5b9ef3891853b83390547b added both classes. `cio-results-filter-and-redo-container` will not technically exist in css but it can be used to preserve backwards compatibility in case folks are using it with selectors